src/main/java/name/valery1707/junit/rule/ConditionalIgnoreRule.java

```diff
@@ -32,16 +37,28 @@
  * @see <a href="https://gist.github.com/rherrmann/7447571">Starting point</a>
  */
 public class ConditionalIgnoreRule implements MethodRule {
+	private static <A extends Annotation> Stream<A> annotation(Annotatable source, Class<A> type) {
+		return Stream
+			.of(source.getAnnotation(type))
+			.filter(Objects::nonNull);
+	}
+
+	private static Stream<ConditionalIgnore> annotations(Annotatable source) {
+		return Stream.concat(
+			annotation(source, ConditionalIgnore.class),
+			annotation(source, ConditionalIgnoreList.class).map(ConditionalIgnoreList::value).flatMap(Stream::of)
+		);
+	}
```

Comment:
\[Checkstyle\] WARNING: 'method def rcurly' has incorrect indentation level 8, expected level should be 2\.